rafthttp/util.go

```diff
@@ -175,3 +175,14 @@ func setPeerURLsHeader(req *http.Request, urls types.URLs) {
 	}
 	req.Header.Set("X-PeerURLs", strings.Join(peerURLs, ","))
 }
+
+// addRemoteFromRequest add remote according to request header
```

Comment:
`// addRemoteFromRequest adds a remote peer according to an http request header`